src/app/teams/teams.component.html

```diff
@@ -13,7 +13,7 @@
     <span class="toolbar-fill"></span>
     <div class="margin-lr-3">
       <button mat-button routerLink='/teams/users' i18n>All Members</button>
-      <button mat-mini-fab (click)="addTeam()" ><mat-icon>add</mat-icon></button>
+      <button mat-mini-fab *ngIf="user.isUserAdmin || userConfig.registrationRequest === 'accepted'" (click)="addTeam()" ><mat-icon>add</mat-icon></button>
```

Comment:
@BhargaviNadendla its not `userConfig.registrationRequest` it should be `user.roles.length`